cmake/Dependencies.cmake

```diff
@@ -128,6 +128,7 @@ if(BUILD_python)
   if(PYTHONLIBS_FOUND AND NUMPY_FOUND AND Boost_PYTHON_FOUND)
     set(HAVE_PYTHON TRUE)
     if(BUILD_python_layer)
+      find_package(Boost 1.46 COMPONENTS regex)
```

Comment:
Thanks for the fix @stas-sl! Actually I didn't test that in my local since Travis test passed. I will modify it soon.